acceptance/bundle/libraries/maven/test.toml

```diff
@@ -0,0 +1,6 @@
+# We run this test only locally for now because we need to figure out how to do
+# bundle destroy on script.cleanup first.
+LocalOnly = true
+
+RecordRequests = true
+FilterRequestsPattern = "POST /api/.*/jobs/create"
```

Comment:
The intention was always to filter out.requests.txt with jq (and then delete the original file), did you try that? I'm curious on how simple or complex that looks in comparison.

Reply:
@denik but `out.requests.txt` is written after the whole script is executed, right? How can I use `jq` to filter it out?

Reply:
You're right, it is indeed written afterwards, I missed that.

I think the server should write requests as they come though, this would help with this case but also with cases where there are multiple commands and you want to examine them separately:

```
$CLI bundle deploy -t dev
mv out.requests.txt out.dev.requests.txt

$CLI bundle deploy -t prod
mv out.requests.txt out.prod.requests.txt
```

Any concerns with this approach @shreyas-goenka ? It should be straighforward to implement.

Reply:
https://github.com/databricks/cli/pull/2368

Reply:
Yeah, I think that makes sense. Since the `out.requests.txt` file is only written to by the testserver, and any reads  of the file would happen when the writes are not, we should not run into any concurrency issues here.

Should we rename this file to something else? Since it's no longer an output file but almost a device associated with the acceptance tests. 

Anytime you want to assert on the requests log you need to for example explicitly record it as an output, and otherwise it's always ignored:
```
mv testserver.requests.txt out.request.txt
```